features/cellular/framework/targets/QUECTEL/BG96/QUECTEL_BG96_CellularNetwork.cpp

```diff
@@ -72,3 +73,44 @@ nsapi_error_t QUECTEL_BG96_CellularNetwork::set_access_technology_impl(RadioAcce
 
     return _at.unlock_return_error();
 }
+
+bool QUECTEL_BG96_CellularNetwork::is_active_context(int *number_of_active_contexts, int cid)
+{
+
+    _at.lock();
+
+    if (number_of_active_contexts) {
+        *number_of_active_contexts = 0;
+    }
+    bool active_found = false;
+    int context_id;
+    int active;
+    // read active contexts
```

Comment:
Can this "QIACT" part be overritten and rest of the code be in base class since it is so much and is copy pasted?